### DISCUSSION TITLE
Sign Out B2C AAD,

### DISCUSSION THREAD

**Discussion by da314pc** [Azure/api-management-developer-portal#2634] [General]
I tried to implement sign out

I get routed to the page https://login.microsoftonline.com/common/oauth2/v2.0/logoutsession

I want to be redirected to the home page after sign out,

I added the home page url to: Front-channel logout URL in B2C

It still gets ignored, 

Also for sign in, with openidconnect

the accesstoken claim mappings get ignored

and im still redirected to the sign up oauth page, even with the first name, last name and email mapped

**Comment by ghost** [comment]
@da314pc, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@da314, we'll see about redirect to home page. Regarding the mapping, can you please copy names of fields in your tokens? Are they given_name, family_name and emails? Something else?

**Comment by ghost** [comment]
@da314pc, we need more information before we start working on this issue. If you prefer to share it in private, please send us an email to apimportalfeedback@microsoft.com with the issue number in its subject.

**Comment by da314pc** [comment]
@azaslonov 

Here are the claim mappings

![image](https://user-images.githubusercontent.com/5604956/132724525-df37d72b-6755-4df3-8b62-926c69faea5a.png)

The claim value of name from the access token is Email,

So I just map the value to all the required fields, but it still fails and redirects to oauth sign up
